Add email with code submitter credentials (#71)
srobo/team-emails: SR2021/2021-01-08-code-submitter-credentials.md

---
to: SR2021 teams
subject: Student Robotics Code Submitter Credentials
---

Hi,

Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.

> The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note what anyone in your team can submit code to the submitter, but only the latest upload will be used.

Reviewer: ```suggestion
The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.
```

> | TLA | Password |

Reviewer: It's probably worth mentioning somewhere that their TLA is their username.

Author: aha, good catch!

---
to: SR2021 teams
subject: Student Robotics Code Submitter Credentials
---

Hi,

Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.

The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.

> When visiting the code submitter, you'll be prompted for a username and password. The username will be your TLA, and the password is:

Reviewer 2: I don't think anyone would try and copy paste a table as a password but maybe better to be explicit?
```suggestion
When visiting the code submitter, you'll be prompted for a username and password. The username will be your TLA, and your password can be found in the table below:
```

| TLA | Password |
| --- | -------- |
| ABC | Test     |

Note that both usernames and passwords are case sensitive.

## Friendlies

> If you're interested in competing in the friendlies this weekend you must let us know beforehand via email or discord, so we know to schedule you some matches. Teams who submit code but haven't signed up won't be entered. These matches are entirely optional, and will not contribute to the league. We'll use a YouTube [livestream](https://youtu.be/iwa-7R6sR8M) to broadcast the matches and as usual we'll provide your logs afterwards. Unlike our other events, these matches will be incredibly relaxed and informal. We encourage anyone to join in the conversation in Discord or in the chat.

Reviewer 2: I don't think our ordinary events are that formal (though they undoubtedly have more structure). Feel free to ignore if you disagree.
```suggestion
If you're interested in competing in the friendlies this weekend you must let us know beforehand via email or discord, so we know to schedule you some matches. Teams who submit code but haven't signed up won't be entered. These matches are entirely optional, and will not contribute to the league. We'll use a YouTube [livestream](https://youtu.be/iwa-7R6sR8M) to broadcast the matches and as usual we'll provide your logs afterwards. These matches will be incredibly relaxed and informal and, as ever, we encourage anyone to join in the conversation in Discord or in the chat.
```

Reviewer 3: :+1:
